### ISSUE TITLE
alert on command "reset solved"

### ISSUE THREAD

**Issue by olsza** [pcottle/learnGitBranching#749]
I suggest adding an alert
when the user enters "reset solved" to display a message like "are you sure?" Y/N
to avoid accidentally deleting a solution to your problems.

**Comment by pcottle** [comment]
Ah yeah, this is a one-way mutation. But just curious, I only used this really for debugging mostly -- how did you come across this command?

**Comment by olsza** [comment]
> Ah yeah, this is a one-way mutation. But just curious, I only used this really for debugging mostly -- how did you come across this command?

Probably in the code, because I translation into Polish and there was information that the "levels were restored by default ..." but I also saw it in the help if you type "show commands"

**Comment by pcottle** [comment]
Leaving this open since yeah, we should perhaps do this. maybe just require `reset solved --confirm`?

**Comment by olsza** [comment]
I think it's a good idea, so by typing ```reset solved``` a message will appear, to reset use the ```--confirm``` parameter to confirm that the levels are restored to default
and if you type ```reset solved --confirm```, it actually restores everything ;)

**Comment by pcottle** [comment]
Yep! I'll leave this open in case someone wants to do an easy PR; if not, I can get to it eventually. Actual work starts up again today 🥺

**Comment by pcottle** [comment]
![Screen Shot 2021-01-04 at 8 39 03 AM](https://user-images.githubusercontent.com/1135007/103551926-52580e80-4e68-11eb-8981-6b15b9319afd.png)
